test/nbrowser/DocTypeConversion.ts

```diff
@@ -0,0 +1,191 @@
+import { UserAPI } from "app/common/UserAPI";
+import { assert, driver } from "mocha-webdriver";
+import * as gu from "test/nbrowser/gristUtils";
+import { setupTestSuite } from "test/nbrowser/testUtils";
+
+describe("Document Type Conversion", function () {
+  this.timeout(20000);
+  const cleanup = setupTestSuite();
+
+  let userApi: UserAPI;
+  let docId: string;
+  let session: gu.Session;
+
+  before(async () => {
+    session = await gu.session().teamSite.login();
+    docId = await session.tempNewDoc(cleanup);
+    userApi = session.createHomeApi();
+  });
+
+  async function assertExists() {
+    await gu.waitToPass(async () => {
+      assert.equal(await editButton.element().getText(), "Edit");
+    });
+    assert.isTrue(await editButton.visible());
+  }
+
+  async function convert(from: String, to: String) {
+    await gu.openDocumentSettings();
+
+    // Check that Document type is from before any conversion was ever apply to It.
+    assert.equal(await displayedLabel.element().getText(), from);
+
+    // Click to open the modal
+    await editButton.click();
+
+    // Wait for modal.
+    await modal.wait();
+
+    let option;
+
+    switch (to) {
+      case "Regular":
+        option = optionRegular;
+        break;
+      case "Template":
+        option = optionTemplate;
+        break;
+      case "Tutorial":
+        option = optionTutorial;
+        break;
+    }
+    // Select the template option
+    await option?.click();
+
+    assert.isTrue(await option?.checked());
+
+    // Confirm the choice
+    await modalConfirm.click();
+
+    // await for the page reload
+    await driver.sleep(200);
```

Comment:
Seems that this value is a little to small for the CI as it pass successfully on my PC.
I will test to increase it to see if it solves the problem.